src/Func.h

```diff
@@ -410,6 +410,14 @@ class FuncRef {
     EXPORT Internal::Function function() const {return func;}
 };
 
+/** Explicit overloads of min and max for FuncRef. These exist to
+ * disambiguate calls to min on FuncRefs when a user has pulled both
+ * Halide::min and std::min into their namespace. */
+// @{
+inline Expr min(FuncRef a, FuncRef b) {return min(Expr(a), Expr(b));}
```

Comment:
Are we doing const-refs for this kind of thing now (the params, not the body of course).